.gitignore

```diff
@@ -1,27 +1,16 @@
-/vendor
-Homestead.yaml
-Homestead.json
-.env
+server/vendor
+server/Homestead.yaml
```

Comment:
I don't see us actually using Homestead with this commit, probably don't need these.